modules/dkan/dkan_datastore/modules/dkan_datastore_api/dkan_datastore_api.module

```diff
@@ -360,7 +360,12 @@ function dkan_datastore_api_services_resources() {
 function _dkan_datastore_api_datastore_index() {
   if ($_SERVER['REQUEST_METHOD'] === 'GET') {
     $params = _dkan_datastore_api_get_params();
+    $cache_hash = md5(json_encode($params));
```

Comment:
@dafeder names are legacy. Remember this is not a completely new implementation but a bypass between the existing code and what we needed for dashboards. 
https://github.com/NuCivic/dkan_datastore/blob/cd5cd595f8fe11f7289cbf1073a1c8f191f2f9e5/modules/dkan_datastore_api/dkan_datastore_api.module#L87

There is a lot of names that need to be changed to reflect the right behavior.  

I can work on renaming this if you want.